core/src/main/bin/questdb.sh

```diff
@@ -44,22 +44,30 @@ function export_pid {
 
 function export_java {
 
-    if [ "$JAVA_HOME" = "" -a -e /usr/libexec/java_home ]; then
-        JAVA_HOME=$(/usr/libexec/java_home -v 1.8)
-    fi
-
-    # check that JAVA_HOME is defined
-    if [ "$JAVA_HOME" = "" ]; then
-        echo "JAVA_HOME is undefined"
-        exit 55
+    # check if this is our runtime
+    if [ -f "$BASE/java" ]; then
+        export JAVA="$BASE/java"
+    else
+      if [ "$JAVA_HOME" = "" -a -e /usr/libexec/java_home ]; then
```

Comment:
Before this change I was able to run `./questdb.sh ...` even when `JAVA_HOME` was not set. The script now makes it mandatory. Should we just fallback to whatever default `java` binary we find on the system as a fallback?

Reply:
Sorry, my mistake, this was not the case before -- this logic was not changed.